### PR TITLE
Added field Show Acknowledgement to Country.

### PR DESCRIPTION
**Pull request by sonnykt**
JIRA: https://digital-engagement.atlassian.net/browse/SDPA-2973

On-behalf-of: @salsadigitalauorg <sonny@salsadigital.com.au>

### CODE REVIEW

File: tide_landing_page.module [hunk 1]
```diff
@@ -71,5 +71,10 @@ function tide_landing_page_form_node_form_alter(&$form, FormStateInterface $form
   if (in_array($form_id, ['node_landing_page_form', 'node_landing_page_edit_form'])) {
     // Change form layout.
     $form['#attached']['library'][] = 'tide_landing_page/landing_page_form';
+
+    // Hide the field Show Acknowledgement to Country if tide_site is missing.
+    if (!\Drupal::moduleHandler()->moduleExists('tide_site')) {
```
Comment (GROwen) [inline tide_landing_page.module]: @sonnykt Why is this required?

Comment (sonnykt) [reply tide_landing_page.module]: @GROwen The "Acknowledgement to Country" field in the Site term is provided by Tide Site. Without Tide Site, the "Show Acknowledgement to Country?" becomes useless and IMO it should be hidden to avoid any confusion.
